blueprints/banking_blueprint/__init__.py

```diff
@@ -0,0 +1,244 @@
+# -*- coding: utf-8 -*-
+"""This module contains a MindMeld application"""
+from mindmeld import Application
+from mindmeld.models import entity_features
+from mindmeld.markup import process_markup
+from mindmeld.core import FormEntity
+import random
+import json
+
+app = Application(__name__)
+
+__all__ = ['app']
+
+#User data and helper functions
+
+user_data = {}
+
+sample_users = {
+    "johndoe123" : 0,
+    "larry_l12" : 1,
+    "splashbro30" : 2,
+}
+
+def _pull_data(request):
+    with open('banking_blueprint/data/sample_user_data.json') as f:
+        global user_data 
+        data = json.load(f)
+        if request.context.get('user_name'):
+            user_name = request.context.get('user_name')
+            user = sample_users[user_name]
+        else:
+            user = random.choice([0, 1, 2])
+
+        user_data = data[user]
+
+def _get(key):
+    return user_data[key]
+
+def _put(key, value):
+    global user_data
+    user_data[key] = value
+
+def check_value(responder):
+    return _get(responder.slots['origin']) >= responder.slots['amount']
+
+#Slot filling forms 
+
+entity_form = {
+    'entities':[
+        FormEntity(
+            entity='account_type',
+            role='origin',
+            responses=['Sure. Transfer from which account?']),
```

Comment:
Can we list the possible accounts here? `'Sure. Transfer from which account - checking or savings?'`